Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-
-    ## Star [/gists/{id}/star]
-    Star resource represents a Gist starred status.
-
-    The Star resource has the following attribute:
-
-    - starred
+The URI for “Question” resource uses a variable component, expressed by [URI Template][]. In this case, we have an id variable as `{question_id}`. Representing this resource is for `/questions/{question_id}` where `question_id` can be substituted for a real `id`.
 
+<a id="uri-parameters"></a>
+### URI Parameters
 
-    + Parameters
+You can describe the parameters in the URI using a list of Parameters. For “Question” it would be as follows:
 
-        + id (string) ... ID of the gist in the form of a hash
+```markdown
++ Parameters
+    + question_id (number) ... ID of the Question in form of an integer
+```
 
-    + Model (application/hal+json)
+The *"question_id"* variable of the URI template is a parameter to every action on this resource. Here defined of an arbitrary type *"number"* and followed by a description for the parameter.
 
-        HAL+JSON representation of Star Resource.
+> Refer to API Blueprint Specification's [Resource Parameters Section][] for additional examples.
 
-        + Headers
+### Actions
 
-                Link: <http:/api.gistfox.com/gists/42/star>;rel="self"
+Similar to before, we're going to describe our action to view a question detail.
 
-        + Body
+```markdown
+### View a question detail [GET]
 
-                {
-                    "_links": {
-                        "self": { "href": "/gists/42/star" },
-                    },
-                    "starred": true
-                }
++ Response 200 (application/json)
 
-    ### Star a Gist [PUT]
-    + Response 204
+            {
+                "question": "Favourite programming language?",
+                "published_at": "2014-11-11T08:40:51.620Z",
+                "url": "/questions/1",
+                "choices": [
+                    {
+                        "answer": "Swift",
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "answer": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "answer": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "answer": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
+            }
+```
 
-    ### Unstar a Gist [DELETE]
-    + Response 204
+#### Response without a body
 
-    ### Check if a Gist is Starred [GET]
-    + Response 200
+You can describe a response without a body, for example a 204 response when we delete our question:
 
-        [Star][]
+```
+### Delete [DELETE]
++ Response 204
+```
 
 ## Complete Blueprint
```

Comment:
I know, as with many things I am commenting here on, this goes on my own head, but: 

- API Blueprint: the name of the language
- API blueprint: a file – document – written in using the API Blueprint language 

"## Complete Blueprint" :arrow_right: "## Complete blueprint"